dvc/remote/gdrive/__init__.py

```diff
@@ -36,7 +36,7 @@ class RemoteGDrive(RemoteBASE):
     REGEX = r"^gdrive://.*$"
     REQUIRES = {"pydrive": "pydrive"}
     GDRIVE_USER_CREDENTIALS_DATA = "GDRIVE_USER_CREDENTIALS_DATA"
-    CREDENTIALS_FILE_PATH = "credentials.json"
+    DEFAULT_USER_CREDENTIALS_FILE = ".dvc/tmp/gdrive-user-credentials.json"
```

Comment:
@MaxRis just to clarify does this file serve as a replacement for the SECTION_GDRIVE_CLIENT_ID and SECTION_GDRIVE_CLIENT_ID ?

Reply:
Disregard the previous comment, please. I see that it is not a replacement.

Reply:
Disregard the comment ^^ please.